src/starbase/process.ts

```diff
@@ -8,6 +8,10 @@ async function executeWithLogging(command: string) {
       childProcess.stdout.pipe(process.stdout);
     }
 
+    if (childProcess.stderr) {
+      childProcess.stderr.pipe(process.stderr);
+    }
+
```

Comment:
This is the main change.  We need to expose stderr so we can see things like JupiterOne synchronization 401 errors when apiKeys expire or are incorrect.